src/test/java/frc/team997/lib/util/DashboardChooserTests.java

```diff
@@ -46,8 +46,10 @@ public void enumDashboardChooserMatchesSendableChooser()
         Field shuffleboardMapField = DashboardChooser.class.getDeclaredField("optionMap");
         shuffleboardMapField.setAccessible(true);
 
+        @SuppressWarnings("unchecked")
```

Comment:
Can we suppress warnings for each function instead? since it's a test and it uses reflection multiple times